go.mod

```diff
@@ -18,7 +18,8 @@ require (
 	github.com/iancoleman/strcase v0.2.0
 	github.com/martinlindhe/base36 v1.1.1
 	github.com/open-policy-agent/opa v0.52.0
-	github.com/pomerium/pomerium v0.20.1-0.20230502181647-150c241f4948
+	github.com/pomerium/csrf v1.7.0
+	github.com/pomerium/pomerium v0.20.1-0.20230523213027-10662d70348b
```

Comment:
I'm also confused why this version tag is based on v0.20.1, when there's a newer v0.22.1 tag. I guess only the datetime and commit hash actually matter though, is that right?